cloud_formation/configs/activities.py

```diff
@@ -142,10 +191,14 @@ def post_init(session, domain):
 
 
 def delete(session, domain):
-    names = AWSNames(domain)
     # DP TODO: delete activities
-    CloudFormationConfiguration('activities', domain).delete(session)
+    #CloudFormationConfiguration('activities', domain).delete(session)
```

Comment:
If you don't delete the Cloud Formation template here it never gets deleted...

Reply:
Good catch.